x-pack/legacy/plugins/apm/mappings.json

```diff
@@ -1,42 +1,734 @@
 {
-  "apm-services-telemetry": {
+  "apm-telemetry": {
```

Comment:
thank you! this is so awesome will be so useful when we add the mapping in the telemetry cluster. Please refer to this file when requesting it :)